pkgs/servers/mail/opensmtpd/default.nix

```diff
@@ -23,7 +23,7 @@ else stdenv.mkDerivation rec {
     sha256 = "291881862888655565e8bbe3cfb743310f5dc0edb6fd28a889a9a547ad767a81";
   };
 
-  patches = [ ./proc_path.diff ];
+  patches = [ ./proc_path.diff ./return-path-fix.diff ];
```

Comment:
Can be fetchpatch used here? Then it is more obvious that we can remove this patch in the next version.

Reply:
Thanks! I didn't know GitHub provided a `.diff` URL for fetching commits, grepping for `fetchpatch` pulled it out. That should be fixed now :)